src/Symfony/Component/Security/Http/ParameterBagUtils.php

```diff
@@ -36,7 +36,7 @@ final class ParameterBagUtils
      *
      * @throws InvalidArgumentException when the given path is malformed
      */
-    public static function getParameterBagValue(ParameterBag $parameters, $path)
+    public static function getParameterBagValue(ParameterBag $parameters, string $path)
```

Comment:
The docblock comments for these can be removed. Perhaps the parameter should be renamed to `$key` if the comment is required otherwise, though I think it can be removed without issues the syntax is documented above it

Reply:
I'd only remove the two now-redundant `@param` type-hints.